src/Eccube/Controller/Admin/Product/CsvImportController.php

```diff
@@ -365,41 +360,31 @@ public function csvProduct(Application $app, Request $request)
                                     }
                                     $ProductClass = $this->createProductClass($row, $Product, $app, $data, $ClassCategory1, $ClassCategory2);
 
-                                    if ($BaseInfo->getOptionProductDeliveryFee() == Constant::ENABLED) {
```

Comment:
@ryo-endo 
when delivery fee disable, we also insert this column into database:
1) admin can edit delivery fee to enable and use this column value
2) when disable this column will not affect to others.
Please confirm this solution. if NG, we will add it back.